src/dmd/id.d

```diff
@@ -304,6 +304,7 @@ immutable Msgtable[] msgtable =
     // For pragma's
     { "Pinline", "inline" },
     { "lib" },
+    { "linkerdirective" },
```

Comment:
Should be `linkerDirective`

Reply:
Really? Like `pragma(startaddress)`? :/

Reply:
So... I feel worse about breaking precedent. Do you really want me to make this change?

Reply:
The startaddress comes from the very beginning of D. Later on, but long ago, we moved to the other convention.